src/AppBundle/Model/User/DTO/CreateUserDTO.php

```diff
@@ -15,6 +15,7 @@
 namespace AppBundle\Model\User\DTO;
 
 use AppBundle\Model\User\User;
+use AppBundle\Validator\Constraints as Captcha;
```

Comment:
please import the class as-is and not a prefix (look at Locale and UniqueProperty)